account_bank_statement_import_online_nordigen/static/src/lib/nordigen-ui/selector.css

```diff
@@ -0,0 +1,209 @@
+
+* {

```

Comment:
This css file is added to `web.assets_backend`. That means it is applied to the whole backend layout, as long as the elements match the selectors. I was going to recommend therefore that you use class names that are a little bit more specific to the module (e.g. `nordigen_institution_*` rather than just `institution_*`).

Moreover, here you are using the universal selector (just like you are using the `html` selector further down). Will that not lead to problems in other parts of the system?

(my web dev skills are notoriously lacking, so let me know if I am missing anything here)